README.md

```diff
@@ -121,6 +112,29 @@ PLUGINS_CONFIG = {
 }
 ```
 
+To enable Gmail API access, there are some common steps for either Service Account and OAuth authentication:
+
+1. Create a **New Project** in [Google Cloud Console](https://console.cloud.google.com/).
+2. Under **APIs and Services**, enable **Gmail API** for the selected project.
+
+**Service Account**
+
+To create a [Service Account](https://support.google.com/a/answer/7378726?hl=en) integration:
+
+3. Still under **APIs and Services**, in **Credentials**, create a new **Service Account** and save the credentials file generated.
+4. With Admin rights, edit the newly created Service Account and expand the **Show Domain-Wide Delegation** section. Enable Google Workspace Domain-wide Delegation and save the changes. Copy the Client ID shown.
+5. With Super Admin rights, open the [Google Workspace admin console](https://admin.google.com). Navigate to **Security**, **API controls**, and select the **Manage Domain Wide Delegation** at the bottom of the page.
+6. Add a new API client and paste in the Client ID copied earlier. In the **OAuth scopes** field add the scopes `https://www.googleapis.com/auth/gmail.readonly` and `https://mail.google.com/`. Save the new client configuration by clicking _Authorize_.
+
+**OAuth**
+
+To create a [OAuth 2.0](https://developers.google.com/identity/protocols/oauth2/web-server) integration:
+
+3. Still under **APIs and Services**, in **Credentials**, create a new **OAuth client ID** selecting the **Web application** application type.
```

Comment:
Does this generate the required credentials file? Step 3 under "Service Account" mentions saving a credentials file, but this step here does not.